packages/server/beaver_triple_service/README.md

```diff
@@ -91,38 +91,48 @@ grpc_health_probe -addr=localhost:54100
 ```
 
 ## JWT token の生成
-YAML ファイルを入力に JWT token を生成します
+YAML ファイルと SECRET_KET を入力に JWT token を生成する．
 
 ### 手順
-btsのイメージを取得します
+#### btsのイメージ取得
 
-  ※ tagはこちらの[ページ](https://github.com/acompany-develop/QuickMPC/pkgs/container/quickmpc-bts)は参照のうえ、latestのtagに変更してください
-  ```
-  docker pull ghcr.io/acompany-develop/quickmpc-bts:<tag>
-  ```
+※ tagはこちらの[ページ](https://github.com/acompany-develop/QuickMPC/pkgs/container/quickmpc-bts)は参照のうえ，latestのtagに変更する．
+```console
+docker pull ghcr.io/acompany-develop/quickmpc-bts:<tag>
+```
 
-JWT tokenの生成
+#### 設定を作成
+[sample設定](https://github.com/acompany-develop/QuickMPC/tree/main/packages/server/beaver_triple_service/cmd/jwt_generator/sample)を参考にして，適当な名前 `<name>` を定めて次の2ファイルを生成する．
+- `<name>.yaml`
+- `server.<name>.env`
 
-  ```console
-  docker run --rm \
-    -v ./path/to/directory_with_config.yml:<mount_path> \
-    ghcr.io/acompany-develop/quickmpc-bts:<tag> ./beaver_triple_service generateJwt \
-    --file /path/to/config.yml \
-    --output <mount_path>                                           # use own configuration
-  ```
+`server.<name>.env`に記載するkeyはBASE64で生成する．
 
---output で指定したPATH直下にクライアントとサーバ向けにそれぞれ `server.<ymlのファイル名>.env`, `client.<ymlのファイル名>.env` というファイル形式の設定ファイルが書き込まれます
+#### JWT tokenを生成
+次のコマンドを実行する．
 
-```
-// server.<ymlのファイル名>.env
-JWT_BASE64_SECRET_KEY=...
+```bash
+docker run --rm \
+  --env-file=/path/to/server.<name>.env \
+  -v /absolute_path/to/directory_with_config.yaml:<mount_path> \
+  ghcr.io/acompany-develop/quickmpc-bts:<tag> ./beaver_triple_service generateJwt \
+  --file <mount_path>/to/<name>.yaml \
+  --output <mount_path>
```

Comment:
Now is it correct that this requires `--env-file`?

Reply:
Yes, it will be required.
Because the environment variable JWT_BASE64_SECRET_KEY must be passed